grub/boot/grub/grub-test.cfg

```diff
@@ -0,0 +1,10 @@
+set timeout=0
+set default=0
+serial --speed=115200 --word=8 --parity=no --stop=1
+terminal_input --append serial
+terminal_output --append serial
+
+menuentry "kernel64" {
+   multiboot /boot/kernel64.bin      integer=42   boolean=1   string=foo booleantwo
```

Comment:
nit: a lot of spaces in between the params

Reply:
Yep, unittesting, to figure out of the space striping works ;)